Tests/Transfer/HyperwalletTransfer.swift

```diff
@@ -114,6 +114,39 @@ class HyperwalletTransferTests: XCTestCase {
         verifyTransferResponse(transferResponse)
         XCTAssertEqual(transferResponse?.token, "trf-123456")
     }
+
+    func testCommitTransfer_success() {
```

Comment:
rename test to testScheduleTransfer_success

Reply:
Renamed